src/services/README.md

```diff
@@ -66,4 +68,154 @@ It is worth noting that HdResponsive component takes a `breakpoints` prop as wel
+const amountOfDays = 1;
+
+const date2 = getNDaysFromDate(initialDate, amountOfDays); // 16.2.2018 at 00:00:00:00
+```
+
+#### convertMsToDays
+
+Returns number of days from miliseconds.
+
+`param {Number} ms: Miliseconds`
+
+```javascript
+const ms = 5 * 24 * 60 * 60 * 1000; // 432000000
+
+const days = convertMsToDays(ms); // 5 days
+```
+
+#### getDaysDateRange
+
+Returns array of date range between 2 dates, including start date and **excluding** end date
+
+`param {Date} startDate: Start of time interval`
+
+`param {Date} endDate: End of time interval`
+
+```javascript
+const startDate = new Date(2018, 0, 1);
+const endDate = new Date(2018, 0, 3);
+
+const timeInterval = getDaysDateRange(startDate, endDate);
+// [ 2018-01-01T00:00:00.000Z, 2018-01-02T00:00:00.000Z ]
+```
+
+#### getIntlDateString
+
+Returns array of **unique** day names in arbitrary(passed) date list
+
+`param {String} locale: Valid locale`
+
+`param {Array} datesArray : List of dates to be checked for day name`
+
+`param {Object} localStringArgs - Locale options` - [Full List](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString#Parameters)
+
+
+```javascript
+const locale = 'en-US';
+const datesArray = [new Date(2018, 0, 1)];
+const localStringArgs = { weekday: 'long' };
+
+const daysInIntervalLong = getIntlDateString(locale, datesArray, localStringArgs);
+// [ 'Monday' ]
+```
+
+#### generateDateCycles
+
+Returns an array of dates in N (`cycleLengthWeeks`) weeks, with M cycles (`amountOfCycles`) from selected offset(in days) from today (`startDateOffset`)
+
+`param {Number} cycleLengthWeeks: Total number of weeks in the cycle`
+
+`param {Number} amountOfCycles: Number of cycles to generate`
+
+`param {Number} startDateOffset: Offset in days e.g.: 0 = today, -1 = yesterday ,...`
+
+```javascript
+const cycleLengthWeeks = 2;
+const amountOfCycles = 2;
+const startDateOffset = 0;
+
+const dateCycles = generateDateCycles(cycleLengthWeeks, amountOfCycles, startDateOffset);
+/*
+  [
+    2020-01-28T00:00:00.000Z,
+    2020-01-29T00:00:00.000Z,
+    2020-01-30T00:00:00.000Z,
+    2020-01-31T00:00:00.000Z,
+    2020-02-01T00:00:00.000Z,
+    2020-02-02T00:00:00.000Z,
+    2020-02-03T00:00:00.000Z,
+    2020-02-04T00:00:00.000Z,
+    2020-02-05T00:00:00.000Z,
+    2020-02-06T00:00:00.000Z,
+    2020-02-07T00:00:00.000Z,
+    2020-02-08T00:00:00.000Z,
+    2020-02-09T00:00:00.000Z,
+    2020-02-10T00:00:00.000Z,
+    2020-02-11T00:00:00.000Z,
+    2020-02-12T00:00:00.000Z,
+    2020-02-13T00:00:00.000Z,
+    2020-02-14T00:00:00.000Z,
+    2020-02-15T00:00:00.000Z,
+    2020-02-16T00:00:00.000Z,
+    2020-02-17T00:00:00.000Z,
+    2020-02-18T00:00:00.000Z,
+    2020-02-19T00:00:00.000Z,
+    2020-02-20T00:00:00.000Z,
+    2020-02-21T00:00:00.000Z,
+    2020-02-22T00:00:00.000Z,
+    2020-02-23T00:00:00.000Z,
+    2020-02-24T00:00:00.000Z
+  ]
+*/
+```
+
+## Utils
+
+#### generateUniqueNumbers
```

Comment:
Can we also mention that an empty or incomplete array is returned because of the sanity checks?

And also that both `min` and `max` are included, or maybe change "open interval" to "closed interval", and make it in bold or something because it's important information.

We should consider changing `getRandomInt` to not include `max`, to match the common reasoning.

Reply:
Great points, will updated. :+1: